### PR TITLE
194 footer component update

### LINKED ISSUE #194: update the Footer component
# acceptance criteria
- [x] linkedIn is added to the `iconsList` object in SocialIcon.jsx
- [x] the 3 section columns in the middle are made smaller
  - the title is size 13
  - the links are size 13
- [x] a link titled "Legal Notices" is placed on the left hand side underneath the copyright info (font-size: 13px)
- [x] copyright info gets reduced to an `h6`
- [ ] the margin top/bottom of the footer is reduced

# story
phenovista has requested a linkedIn icon be added to their footer's list of social icons. while we're updating that component, we'd like to add a few more changes as well.

# screenshots
this was done using the inspect tool, so use it as a reference only
image

### PR DESCRIPTION
**Pull request by summer-cook**
# Story
- Update the footer component styling
- add a linkedin icon to the SocialIcon component for user in the footer

# Screenshot
image

# Acceptance
- [ ] linkedIn is added to the `iconsList` object in SocialIcon.jsx
- [ ] the 3 section columns in the middle are made smaller
  - the title is size 13
  - the links are size 13
- [ ] a link titled "Legal Notices" is placed on the left hand side underneath the copyright info (font-size: 13px)
- [ ] copyright info gets reduced to an `h6`
- [ ] the margin top/bottom of the footer is reduced

### CODE REVIEW

**Comment by summer-cook** [comment]
@alishaevn do you have a subscription to Flaticon pro? I was not able to find an icon that is the exact same as the pre-existing ones because flaticon does not let you download SVGs without premium, and finding a circle linkedin svg with transparent lettering elsewhere was tricky. the "in" in the linkedin has a fill, which is not ideal, so it will always be black even if on a different background whereas the others are all transparent. I suppose we could pass it the background color of the footer but if you have a flaticon login that would be easier

**Comment by summer-cook** [comment]
also the acceptance criteria of 
 - [ ] the margin top/bottom of the footer is reduced
 would probably need to be done in the webstore. it does not have margin/would need to be given margin on import to the home page. I reduced the top/bottom padding slightly inside the footer though to go along with the smaller text sizes

File: src/compounds/Footer/Footer.jsx [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 
 const Footer = ({ companyName, sections, socials, color, ...props }) => (
-  <footer id='footer' className={`container-fluid bg-primary py-4 text-${color}`}>
+  <footer id='footer' className={`container-fluid bg-primary py-3 text-${color}`}>
     <div className='container'>
       <div className='row'>
         <div className='col-12 col-md-3 text-center text-md-start'>
-          <h5>© {companyName} {new Date().getFullYear()}</h5>
+          <h6>© {companyName} {new Date().getFullYear()}</h6>
+          <Link label='Legal Notices' href='/legal-notices' className={`text-${color} link-group-links text-decoration-none`}/>
```
Comment (summer-cook) [inline src/compounds/Footer/Footer.jsx:13]: i also wasn't sure what this href should be. i didn't see a page for it in the webstore yet

Comment (alishaevn) [reply src/compounds/Footer/Footer.jsx:13]: `legal-notices` is correct!

**Comment by alishaevn** [comment]
> @alishaevn do you have a subscription to Flaticon pro? I was not able to find an icon that is the exact same as the pre-existing ones because flaticon does not let you download SVGs without premium, and finding a circle linkedin svg with transparent lettering elsewhere was tricky. the "in" in the linkedin svg has a fill, which is not ideal, so it will always be black even if on a different background whereas the others are all transparent. I suppose we could pass it the background color of the footer but if you have a flaticon login that would be easier. i guess as a last resort we could edit it on illustrator or something.

hmmmm. I don't have a pro subscription either. let me see if I can find anything though.

**Comment by alishaevn** [comment]
#194

**Comment by alishaevn** [comment]
re: the linked in svg

I wound up downloading a png and converting it to an svg. (see below). I also use https://react-svgr.com/playground/ to convert an svg to a react component. (in case you didn't have some other way you use, or it's a difficult way)

<details>
<summary>open this</summary>

linked-in
</details>

**Comment by summer-cook** [comment]
> re: the linked in svg
> 
> I wound up downloading a png and converting it to an svg. (see below). I also use https://react-svgr.com/playground/ to convert an svg to a react component. (in case you didn't have some other way you use, or it's a difficult way)
> 
> open this

great, that worked perfectly! 
image

File: src/assets/img/thank-you.md [hunk 2]
```diff
@@ -11,3 +11,4 @@ Our icons were sourced from the following:
 * [Twitter social badge icons created by Freepik - Flaticon](https://www.flaticon.com/free-icons/twitter-social-badge)
 * [Vimeo icons created by Md Tanvirul Haque - Flaticon](https://www.flaticon.com/free-icons/vimeo)
 * [Youtube icons created by Freepik - Flaticon](https://www.flaticon.com/free-icons/youtube)
+* [Linkedin icons created by IconScout](https://iconscout.com/free-icon/linkedin-48)
```
Comment (alishaevn) [inline src/assets/img/thank-you.md:14]: what's icon scout? the png I converted came from <a href="https://www.flaticon.com/free-icons/mass-media" title="mass media icons">Mass media icons created by mohammed mahdi - Flaticon</a>

File: src/assets/svg/Linkedin.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+
+const Linkedin = ({ height, fill }) => (
+  <svg
+    height={height}
+    width={height}
+    fill={fill}
+    viewBox="0 0 144 144"
```
Comment (alishaevn) [inline src/assets/svg/Linkedin.jsx:8]: can you change all the quotes here to singles? surprised lint didn't catch it. or maybe it's not checking these files.

File: src/assets/svg/Linkedin.jsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+
+const Linkedin = ({ height, fill }) => (
```
Comment (alishaevn) [inline src/assets/svg/Linkedin.jsx:3]: suuuuuuuuuper nit picky, but can this file and variable name be changed to `LinkedIn`?

**Review by alishaevn — CHANGES_REQUESTED** [review]

File: src/assets/img/thank-you.md (hunk 2 above)
Comment (summer-cook) [reply src/assets/img/thank-you.md:14]: ooops i forgot to change this as well! the one from before was from  icon scout.

**Review by alishaevn — APPROVED** [review]

**Pull request merged**